### PR TITLE
nho: trailblazes wdi8

### PR DESCRIPTION
**Pull request by nolds9**
need to iron out: 
- schedule
- mini-lessons
- outcomes
- attendance 
- one-on-ones?
- genius bars?
- technical requirements?

Would love ideas on tier breakdowns in `evaluation.rb`

### CODE REVIEW

**Comment by nolds9** [comment]
schedule -> nick to propose initial schedule
outcomes -> check Roadmap
attendance -> strict
one-on-ones -> regular
genius bar -> propose solutions
technical requirements -> group to evaluate

**Comment by RobertAKARobin** [comment]
Re: Genius Bar/1on1s: I'd say use the Genius Bar schedule from P1, and we'll just keep on truckin' with 1on1s as normal. On Thursday we can discuss whether we'd like to do Genius Bar tokens, but that shouldn't necessitate changing the schedule.

File: evaluation.md [hunk 1]
```diff
@@ -1,53 +1,9 @@
...
+| Planning / Process / Submission | No submission received, or submission does not contain basic documentation of planning (ERD, wireframes, commit history, etc) | App is submitted, with basic evidence of planning. Documentation exists, but lacks common areas such as setup instructions, description of application functionality and link to deployed application | Submission contains clear evidence of planning, adequate documentation, include all from previous category, as well as additional information such as unsolved issues. | Submission includes everything in previous category, as well as evidence of  planning tools, such as Trello, and, incorporates workflows such as feature branching, code review, github issue / user story tracking, and justification of technical decisions. |
```
Comment (jshawl) [inline evaluation.md:9]: :+1:

**Comment by jshawl** [comment]
:+1:

**Pull request merged**